File: prime.java

class primeNO {
    static boolean isPrime(int n)
    {
       
        if (n <= 1)
            return false;

  
        for (int i = 2; i < n; i++)
            if (n % i == 0)
                return false;

        return true;
    }

   
    public static void main(String args[])
    {
        if (isPrime(11))
            System.out.println(" true");
        else
            System.out.println(" false");
        if (isPrime(15))
            System.out.println(" true");
        else
            System.out.println(" false");
    }
}

File: prime.py
def is_prime(num):
    if num <= 1:
        return False
    for i in range(2, int(num**0.5) + 1):
        if num % i == 0:
            return False
    return True

num = 11
if is_prime(num):
    print(f"{num} is a prime number")
else:
    print(f"{num} is not a prime number")
